### PR TITLE
main readme update

### PR DESCRIPTION
**Pull request by lack**
- **Consistent naming**
- **Add sytem-test build badge**
- **Update install instructions**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
-# kube-compare
+# cluster-compare
```
Comment (natifridman) [inline README.md:1]: It's a bit confusing. The repo name is `kube-compare` and here we are calling it `cluster-compare`

Comment (natifridman) [reply README.md:1]: I think it's usually the name of the repo

Comment (lack) [reply README.md]: Put that back.

**Pull request merged**